tiledb/api/c_api/config/config_api_external.h

```diff
@@ -458,6 +458,10 @@ TILEDB_EXPORT void tiledb_config_free(tiledb_config_t** config) TILEDB_NOEXCEPT;
  *    The scale factor for exponential backoff when connecting to S3.
  *    Any `long` value is acceptable. <br>
  *    **Default**: 25
+ * - `vfs.s3.custom_headers.` <br>
```

Comment:
Minor tweak to make this `vfs.s3.custom_headers.*`. The `*` to hopefully help indicate an "anything" there.